e2e-tests/oauth-flow/rfc021/do-test.sh

```diff
@@ -94,10 +115,35 @@ fi
 
 DPOP=$(cat ./node-B/dpop.txt)
 
+# Introspect access token with a form post
+echo "------------------------------------"
+echo "Introspect access token..."
+echo "------------------------------------"
+RESPONSE=$(curl -X POST -s --data "token=$ACCESS_TOKEN" http://localhost:18081/internal/auth/v2/accesstoken/introspect)
+echo $RESPONSE
+# Check that it contains "active": true
+if echo $RESPONSE | grep -q "active.*true"; then
+  echo "access token is active"
+else
+  echo "FAILED: Access token is not active" 1>&2
+  echo $RESPONSE
+  exitWithDockerLogs 1
+fi
+# Check that it contains "employee_name":"John Doe"
+if echo $RESPONSE | grep -q "employee_name.*John Doe"; then
+  echo "employee_name claim is present"
+else
+  echo "FAILED: kissing/invalid employee_name" 1>&2
```

Comment:
`kissing`?

Reply:
Weird, I think Github Copilot generated this...